### DISCUSSION TITLE
Does Vitest-UI display the test failure location and reason in the Code tab?

### DISCUSSION THREAD

**Discussion by elliothimmelfarb** [vitest-dev/vitest#2269] [Q&A]
Just finished integrating Vitest into our project, very pleased with it so far. It is blazing fast and feature-rich!

Been using Vitest-UI but can't seem to get something working. In the example image seen in the docs [here](https://vitest.dev/guide/ui.html), it appears that a failed test renders the failure location and reason inline in the code tab. Here is the image:

![inline error reporting](https://user-images.githubusercontent.com/11247099/171992272-7c6057e2-80c3-4b17-a7b6-0ac28e5a5e0b.png)

Is this working for anyone else? I can't seem to get this working locally for my team.

Thank you!

**Comment by elliothimmelfarb** [comment]
I've got an update to this. It appears I can get this working with inline source tests, but it still doesn't work for other types of tests.

**Reply by bauerindustries** [reply]
I having the same issues, both in the CLI and in the UI. I have run my code in the StackBlitz online sandbox, and I get the code highlighting/snippets, showing exactly where the error is occurring. But not locally 😔

UPDATE/FIX: I removed spaces in the folder names that contained the project, and the code snippets were displayed in the test results. I remembered that I experienced a similar issue with special characters in folder names on the Typescript course... 🤦‍♂️

**Comment by 51fe** [comment]
Try vitest CLI
